interfaces/vrep/robots/LBR4pVrepRobot.m

```diff
@@ -0,0 +1,69 @@
+classdef LBR4pVrepRobot < DQ_VrepRobot
+    %UNTITLED Summary of this class goes here
```

Comment:
Just for the sake of brevity, I'll indicate the necessity of documentation by just writing _doc_ from now on. 😃